### PR TITLE
mitosheet: add isEnterprise designation

### PR DESCRIPTION
**Pull request by naterush**
# Description

Implements: https://www.notion.so/trymito/Mito-Enterprise-User-Profile-edab23285c214322ad54d59d01a350fb?pvs=4

# Testing

Try:
1. Enterprise and not pro through the user profile
2. Enterprise and pro through the user profile
3. Pro and not enterprise through user profile
4. Turning on Snowflake with Pro only through config
5. Turning on Enterprise with helper package.

# Documentation

Probably fine for now, we can document enterprise config.

### CODE REVIEW

File: mitosheet/mitosheet/enterprise/mito_config.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 MITO_CONFIG_CODE_SNIPPETS_VERSION  = 'MITO_CONFIG_CODE_SNIPPETS_VERSION' 
 MITO_CONFIG_CODE_SNIPPETS_URL = 'MITO_CONFIG_CODE_SNIPPETS_URL'
 MITO_CONFIG_DISABLE_TOURS = 'MITO_CONFIG_DISABLE_TOURS'
+MITO_CONFIG_FEATURE_ENABLE_SNOWFLAKE_IMPORT = 'MITO_CONFIG_FEATURE_ENABLE_SNOWFLAKE_IMPORT'
```
Comment (naterush) [inline mitosheet/mitosheet/enterprise/mito_config.py:20]: I think we could refactor this file. Currently, adding a single boolean flag takes adding it in 3-5 places, depending on how you could. Ideally, we'd do it in less than that...

**Comment by naterush** [comment]
@aarondr77 I'm having trouble deploying the enterprise helper. Can you try doing so (and then sharing package ownership with me)?

**Comment by aarondr77** [comment]
Screenshot 2023-02-18 at 1 07 41 PM

**Comment by aarondr77** [comment]
This shows the Mito Enterprise message, but the feature is still accessible. 

Screenshot 2023-02-18 at 1 10 01 PM

**Comment by aarondr77** [comment]
When the user is on Mito Enterprise and they click on the Mito Enterprise button, we should update this to show them Mito Enterprise stuff

Screenshot 2023-02-18 at 1 11 55 PM

File: mitosheet-helpers/mitosheet_helper_pro/LICENSE [hunk 2]
```diff
@@ -1,4 +1,4 @@
-The Mito Pro and Enterprise license (the “Enterprise License”)
+The Mito Pro and Enterprise license (the “Pro and Enterprise License”)
```
Comment (aarondr77) [inline mitosheet-helpers/mitosheet_helper_pro/LICENSE:1]: In the mitosheet_helper_pro package, it should only be the Pro liscense, right?

File: mitosheet/mitosheet/user/schemas.py [hunk 3]
```diff
@@ -247,7 +248,34 @@
     UJ_RECEIVED_CHECKLISTS: {}
 }
 
+"""
+Adds UJ_MITOSHEET_ENTERPRISE and sets it equal to Falase
```
Comment (aarondr77) [inline mitosheet/mitosheet/user/schemas.py:252]: Falase -> False

File: mitosheet/src/components/elements/MitoProUpgradePrompt.tsx [hunk 4]
```diff
@@ -2,23 +2,25 @@ import React from 'react';
...
     message: string | undefined;
+    proOrEnterprise: 'Pro' | 'Enterprise'
 }): JSX.Element => {
+
     return (
         <div>
             <Row justify='space-between' align='center'>
                 <p className='text-body-1'>
-                    {props.message || 'This is a Mito Pro feature. To access all Mito Pro functionality, please upgrade.'}
+                    {props.message || `This is a Mito ${props.proOrEnterprise} feature. To access all Mito Pro functionality, please upgrade.`}
```
Comment (aarondr77) [inline mitosheet/src/components/elements/MitoProUpgradePrompt.tsx:14]: If props.proOrEnterprise is Enterprise, then we should update the message to "To access all Mito Enterprise functionality, please upgrade"

**Review by aarondr77 — COMMENTED** [review]
Some small cleanup, then LGTM

File: mitosheet-helpers/mitosheet_helper_enterprise/pyproject.toml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mitosheet_helper_enterprise"
+version = "0.0.3"
```
Comment (aarondr77) [inline mitosheet-helpers/mitosheet_helper_enterprise/pyproject.toml:7]: This version does not match the version listed in `mitosheet-helpers/mitosheet_helper_enterprise/setup.cfg`

**Comment by aarondr77** [comment]
You're invited to be an owner of `mitosheet-helper-enterprise` on both pypi and testpypi

**Pull request merged**